File: app/src/main/java/meugeninua/appfromscratch/ui/activities/main/fragment/main/binding/MainBinding.java
package meugeninua.appfromscratch.ui.activities.main.fragment.main.binding;

import meugeninua.appfromscratch.ui.activities.base.fragments.base.binding.Binding;
import meugeninua.appfromscratch.ui.activities.main.fragment.main.view.MainView;

public interface MainBinding extends Binding {

    void displayTime();

    void setupListeners(MainView view);

    void enableButtons(boolean started);
}

File: app/src/main/java/meugeninua/appfromscratch/ui/activities/main/fragment/main/binding/MainBinding.kt
package meugeninua.appfromscratch.ui.activities.main.fragment.main.binding

import meugeninua.appfromscratch.ui.activities.base.fragments.base.binding.Binding
import meugeninua.appfromscratch.ui.activities.main.fragment.main.view.MainView

interface MainBinding : Binding {

    fun displayTime()

    fun setupListeners(view: MainView)

    fun enableButtons(started: Boolean)
}
